modules/wizard/view-wizard-page.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * Displays the wizard
+ *
+ * @since 2.7
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Don't access directly.
+};
+$admin_body_class[] = 'pll-wizard';
+$admin_body_class[] = 'wp-core-ui';
+if ( function_exists( 'is_rtl' ) && is_rtl() ) {
+	$admin_body_class[] = 'rtl';
+}
+?>
+<!DOCTYPE html>
+<html <?php language_attributes(); ?>>
+	<head>
+		<meta name="viewport" content="width=device-width" />
+		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+		<title>
+		<?php
+		printf(
+			/* translators: %s is the plugin name */
+			esc_html__( '%s &rsaquo; Setup', 'polylang' ),
+			esc_html( POLYLANG )
+		);
+		?>
+		</title>
+		<script type="text/javascript">
+			var ajaxurl = '<?php echo esc_url( admin_url( 'admin-ajax.php', 'relative' ) ); ?>';
+		</script>
+		<?php do_action( 'admin_enqueue_scripts' ); ?>
+		<?php wp_print_scripts( $this->steps[ $this->step ]['scripts'] ); ?>
+		<?php wp_print_styles( array_merge( $this->styles, $this->steps[ $this->step ]['styles'] ) ); ?>
+		<?php do_action( 'admin_head' ); ?>
+	</head>
+	<!-- <body class="pll-wizard wp-core-ui"> -->
+	<body class="<?php echo join( ' ', array_map( 'sanitize_key', $admin_body_class ) ); ?>">
```

Comment:
The sanitization wasn't really necessary as all classes are hardcoded. But I guess it makes WPS happy ;-). You can remove the comment though.

Reply:
Yes. It's for WPCS reasons: WordPress.Security.EscapeOutput.OutputNotEscaped
Indeed, now $admin_body_class isn't filterable and no external class can be added.
And I don't it will be possible in the future. At least the code will be ready here.